.github/CODEOWNERS

```diff
@@ -0,0 +1,47 @@
+# Default code owners when not specified.  
+# We prefer to explicitly specify owners for all subdirectories.
+*                                warner, dckc, Chris-Hibbert, FUDCo, michaelfig
+
+/* @Agoric/codeowner-tooling    # Toplevel files only
+/.github/                        kriskowal, michaelfig, erights
+/docker/                         kriskowal, michaelfig, erights
+/patches/                        kriskowal, michaelfig, erights
+/scripts/                        kriskowal, michaelfig, erights
+/yarn.lock                       mhofman
+
+# Golang parts of the repo.
+/go.*                            JimLarson
+/golang/                         JimLarson
+
+# Explicit owners for our packages
+/packages/ERTP/                  erights, hibbert
+/packages/SwingSet/              warner, FUDCo
+/packages/access-token/          kriskowal, michaelfig
+/packages/agoric-cli/            kriskowal, michaelfig
+/packages/assert/                kriskowal, turadg, michaelfig, erights
+/packages/cosmic-swingset/       michaelfig
+/packages/deploy-script-support/ michaelfig
+/packages/deployment/            michaelfig
+/packages/eslint-config/         kriskowal, michaelfig, erights, turadg, mhofman
+/packages/governance/            hibbert
+/packages/import-manager/        michaelfig
+/packages/notifier/              kriskowal, turadg, michaelfig, erights
+/packages/pegasus/               michaelfig
+/packages/run-protocol           hibbert, turadg
+/packages/same-structure/        kriskowal, turadg, michaelfig, erights
+/packages/sharing-service/       michaelfig, hibbert
+/packages/solo/                  kriskowal, michaelfig
+/packages/sparse-ints/           kriskowal, turadg, michaelfig
+/packages/spawner/               michaelfig
+/packages/stat-logger/           warner, FUDCo
+/packages/store/                 warner, FUDCo, erights
```

Comment:
If there's any suggestion of priority here, I'd put `erights` first, then `FUDCo`, then me.. I've barely touched that module.

Reply:
I don't think there is a priority, I think if no reviewer explicitly assigned, it'll ask all to review.